src/cloud-formation/deploy-dev.yaml

```
@@ -1,28 +1,28 @@
 ---
 parameters:
-  BrokerImageVersion: '0.7.0'
-  EnableWafRulesGeneral: 'true'
+  BrokerImageVersion: '0.8.0'
+  DeployAlb: 'true'
+  DeployAlbDnsRecord: 'true'
+  DeployBastionHost: 'true'
+  DeployBroker: 'true'
+  DeployContainers: 'true'
+  DeployDb: 'true'
+  DeployNlb: 'true'
+  DeployNlbVpcLink: 'true'
+  DeployPostgrest: 'true'
+  DeployProcessor: 'true'
+  DeployRestApi: 'true'
+  DeployRestApiDnsRecord: 'true'
+  DeployRouteTables: 'true'
+  DeployStack: 'true'
+  DeployVpc: 'true'
+  DeployWaf: 'false'
+  EnableWafRulesGeneral: 'false'
   EnableWafRulesRestApi: 'false'
   EnableWafRulesWebSocket: 'false'
-  Environment: 'dev'
-  MaybeDeployAlb: 'true'
-  MaybeDeployAlbDnsRecord: 'true'
-  MaybeDeployBastionHost: 'true'
-  MaybeDeployBroker: 'true'
-  MaybeDeployContainers: 'true'
-  MaybeDeployDb: 'true'
-  MaybeDeployNlb: 'true'
-  MaybeDeployNlbVpcLink: 'true'
-  MaybeDeployPostgrest: 'true'
-  MaybeDeployProcessor: 'true'
-  MaybeDeployRestApi: 'true'
-  MaybeDeployRestApiDnsRecord: 'true'
-  MaybeDeployRouteTables: 'true'
-  MaybeDeployStack: 'true'
-  MaybeDeployVpc: 'true'
-  MaybeDeployWaf: 'true'
+  Environment: 'ECO-2188'
   Network: 'testnet'
-  ProcessorImageVersion: '0.5.0'
+  ProcessorImageVersion: '0.5.9'
```

```suggestion
  ProcessorImageVersion: '0.6.0'
```

You can use `0.6.0`, it's technically more up to date than `0.5.9` now!